apps/site/src/views/Schedule/components/Countdown/Countdown.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import Col from "react-bootstrap/Col";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+
+import CountdownItem from "../CountdownItem/CountdownItem";
+
+const COUNTDOWN_FROM = "11/4/2023 10:00:00";
+
+const SECOND = 1000;
+const MINUTE = SECOND * 60;
+const HOUR = MINUTE * 60;
+const DAY = HOUR * 24;
+
+export default function Countdown() {
+	const [remaining, setRemaining] = useState({
+		days: 0,
+		hours: 0,
+		minutes: 0,
+		seconds: 0,
+	});
+
+	useEffect(() => {
+		const timer = setInterval(handleCountdown, 1000);
+
+		return () => clearInterval(timer);
+	}, []);
+
+	const handleCountdown = () => {
+		const end = new Date(COUNTDOWN_FROM);
+
+		const now = new Date();
+
+		const distance = +end - +now;
+
+		const days = Math.floor(distance / DAY);
+		const hours = Math.floor((distance % DAY) / HOUR);
+		const minutes = Math.floor((distance % HOUR) / MINUTE);
+		const seconds = Math.floor((distance % MINUTE) / SECOND);
+
+		setRemaining({
+			days,
+			hours,
+			minutes,
+			seconds,
+		});
```

Comment:
Nitpick: having separate lines is helpful in some cases, but I think this could be reduced to one line.